app/controllers/candidate_interface/work_history/edit_controller.rb

```diff
@@ -14,18 +14,20 @@ def new
                                   start_date_year: start_date.year,
                                   end_date_month: end_date&.month || '',
                                   end_date_year: end_date&.year || '',
-                                  add_another_job: true,
+                                  do_not_add_another_job: true,
```

Comment:
Any reason not to use `add_another_job: false`?

Reply:
Yeah that would be ideal, however we now need to do reverse the logic in order to pre-select the correct radio button.
See https://github.com/DFE-Digital/apply-for-teacher-training/pull/3478

Reply:
Not sure I follow… `add_another_job: false` seems much clearer. Can we not reverse the boolean on the form?

Reply:
@paulrobertlloyd The radio button needs to resolve to `true` in order to be selected. In response to the question 'Do you want to add another job?' we want 'No' to be selected, not 'Yes' right?

Reply:
@TobyRet can't we just initialise the form with the `no` option selected by default, via `WorkExperienceForm.new(add_another_job: false)`?

Reply:
👍 Yep no problem @frankieroberto. I've made the changes